sentinelhub/__init__.py

```diff
@@ -54,6 +54,7 @@
 )
 from .data_collections import DataCollection
 from .data_collections_bands import Band, Unit
+from .data_utils import get_failed_requests, statistical_to_dataframe
```

Comment:
I am not sure we'd want to add this to the global `__init__`, especially since it requires pandas. I think it's completely fine if users have to import it directly with `from sentinelhub.data_utils import ...`